lib/tasks/fix_existing_organogram_filenames.rake

```diff
@@ -0,0 +1,6 @@
+require_relative 'update_organogram_filenames.rb'
+
+desc "Fix existing organogram filenames"
+task :fix_existing_organogram_filenames => :environment do
+  UpdateOrganogramFilenames.new().call
```

Comment:
Generally in Ruby if a function doesn't take any arguments you would call it without the `()`.

```suggestion
  UpdateOrganogramFilenames.new.call
```